app/models/order.rb

```diff
@@ -29,18 +28,4 @@ def final_total
       self.total_cost = 0
     end
   end
-
-  private
-
-  def calculate_new_order_total(_loan)
-    calculate_order_total
-  end
-
-  def calculate_order_total
-    if loans.present?
-      self.total_cost = loans.map(&:amount).reduce(:+)
-    else
-      self.total_cost = 0
-    end
-  end
```

Comment:
I was using these with before_add and after_add filters that weren't working, so I just changed to have a method final_total instead.